lib/util/exec/index.ts

```diff
@@ -34,7 +37,7 @@ export async function setExecConfig(
     execConfig[key] = value || null;
   }
   if (execConfig.binarySource === 'docker') {
-    await removeDanglingContainers();
+    await removeDanglingContainers(getContainerLabel(config.dockerChildPrefix));
```

Comment:
Could this logic be pushed into the removeDanglingContainers() logic instead?

Reply:
We can do that, but we need access to the config somehow. If we don't pass it in, we need to make it global somehow.

Reply:
```suggestion
    await removeDanglingContainers(config.dockerChildPrefix);
```
I think @rarkins  prefer something like this. 🙃